snippets/maintainDigits.md

```diff
@@ -0,0 +1,14 @@
+### setDigits
```

Comment:
This is an extremely confusing method and doesn't provide any real value, as you can achieve the exact same thing with native methods. The returned result could be easily unusable in many situations and there could be tons of problems using this in anything (try passing an array as input - `setDigits([10,2],5,1)` returns `"010,2"` which is highly confusing and should not work at all). I'm voting to remove the snippet entirely and move on.

Reply:
You're right. I wrote it in mind with maintaining digit length, for example in the colon time format. Like I'm mentioning above these were written dependent one another in some ways. This can also be used to maintain digit length of a real price rather than having it rounded to the nearest tenth cent.

Just wanted to throw them all and see what stuck haha. Thanks for your review everyone.